src/http/api/account.ts

```diff
@@ -134,10 +134,7 @@ export function createGetAccountFollowsHandler(
                 return new Response(null, { status: 400 });
             }
 
-            const account = await accountRepository.getByApId(new URL(apId));
-            if (!account) {
-                return new Response(null, { status: 400 });
-            }
+            const account = await accountRepository.getByApId(new URL(apId)) as PersistedAccount;
```

Comment:
We shouldn't cast to a `PersistedAccount` here - because the value can be `null`